src/Controller/DefaultController.php

```diff
@@ -161,6 +164,8 @@ public function voteAction(Request $request)
 
         $yourVote = $githubUserId ? (int) $this->redis->zscore('rfc/' . $rfc->id, $githubUserId) : 0;
 
+        $this->mercurePublisher->publish('vote', ['rfc' => $rfc->id]);
```

Comment:
Is it too heavy to send the JSON payload through Mercure too? It could avoid the extra calls to `/vote`.

Reply:
Babysteps :-) Will do that next